src/main.rs

```diff
@@ -32,59 +32,69 @@ struct Args {
     flag_max_line_length: bool,
 }
 
-fn main() {
-    let args: Args = Docopt::new(USAGE)
-        .and_then(|d| d.decode())
-        .unwrap_or_else(|e| e.exit());
-
-    // just for now.
-    let filename = args.arg_file.unwrap();
-    let file = File::open(&filename).ok().expect("I couldn't open that file, sorry :(");
-    let reader = BufReader::new(file);
+struct CountResults {
+    bytes: usize,
```

Comment:
Iterator::count() and String::len() return usize, so anything else will require casts.